crates/cext/src/circuit.rs

```diff
@@ -1151,3 +1152,98 @@ pub unsafe extern "C" fn qk_circuit_delay(
 
     ExitCode::Success
 }
+
+/// @ingroup QkCircuit
+/// Convert a given circuit to a DAG.
+///
+/// @param circuit A pointer to the circuit from which to create the DAG.
+/// @param copy_operations Performs a deep copy of operations if set to true
```

Comment:
Flagging this to be removed along with param as mentioned in @jakelishman's comment.

Reply:
Done